scripts/serve.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import SimpleHTTPServer
+
+class MyHTTPRequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+    def end_headers(self):
+        self.send_my_headers()
+        SimpleHTTPServer.SimpleHTTPRequestHandler.end_headers(self)
+
+    def send_my_headers(self):
```

Comment:
Use a clearer name -- send_no_cache_headers maybe. Or why not just inline this under end_headers()?

Reply:
`end_headers()` now has `send_my_headers()` contents. Removed `send_my_headers()`!